templates/layouts/layout.html

```diff
@@ -3,12 +3,15 @@
 <html lang="en">
 
 <head>
-    <meta charset="utf-8">
```

Comment:
` <meta charset="utf-8">` must come first and the the order in which other tags comes in matter too I'm afraid.

Reply:
🤦‍♂️ Well, that was a bit stupid on my part... (I removed the restructuring)

Reply:
No worries it happens to all of us :-)